katran/lib/KatranLb.cpp

```diff
@@ -39,6 +39,8 @@ constexpr int kFirstElem = 0;
 constexpr int kError = -1;
 constexpr uint32_t kMaxQuicId = 65535; // 2^16-1
 constexpr folly::StringPiece kEmptyString = "";
+constexpr uint64_t kEnabled = 1;
+constexpr uint64_t kDisabled = 1;
```

Comment:
enabled and disabled have the same value.
Thoughts on making them an enum instead?
For example, something along the lines of: 
`
enum IntospectionControl : uint8_t { ENABLED, DISABLED} 
`

Reply:
ops. missed during merge. let me fix it in a few